L1Trigger/TrackerTFP/src/KalmanFilter.cc

```diff
@@ -215,8 +225,6 @@ namespace trackerTFP {
         state = pop_front(stack);
       streamOutput.push_back(state);
       // The remainder of the code in this loop deals with combinatoric states.
-      //if (!state || !state->stub() || state->layer() != layer_)
```

Comment:
these needs to be deleted for cmssw

Reply:
You say I shall delete a line I have already deleted?